### PR TITLE
 WINC-803: Update correct WMCO subscription info

### PR DESCRIPTION
**Pull request by selansen**
This commit aims to address [WINC-803](https://issues.redhat.com//browse/WINC-803) issue. We need to update
 OpenShift Container Platform for Windows Containers subscription
 as subscription info to be in sync with rlease and support docs.

Signed-off-by: selansen <esiva@redhat.com>

### CODE REVIEW

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -27,12 +27,13 @@ spec:
     or by specifying existing instances through a [ConfigMap](https://docs.openshift.com/container-platform/latest/windows_containers/configuring-byoh-windows-instance.html)
```
Comment (jrvaldes) [inline config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:27]: @selansen , You may need to run `make bundle` to reflect the changes in the bundle.

Comment (selansen) [reply config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:27]: why do I need to run make bundle? Looked into previous changes in the same file, haven't noticed anyone ran make bundle for this file change. Am I missing anything?

Comment (selansen) [reply config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:27]: I am not creating any bundle image.

Comment (jrvaldes) [reply config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:27]: > why do I need to run make bundle? Looked into previous changes in the same file, haven't noticed anyone ran make bundle for this file change. Am I missing anything?

`make bundle` will apply the changes in the bundle CSV, [here](https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L36). Otherwise won't take effect.

For references, you can take a look at https://github.com/openshift/windows-machine-config-operator/pull/780.

Comment (jrvaldes) [reply config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:27]: > This will be done as part of release prep, right?

Yes, and running `make bundle` now will apply the changes in the [bundle CSV](https://github.com/openshift/windows-machine-config-operator/blob/master/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L36).

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -11,9 +11,8 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
```
Comment (jrvaldes) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: PR title should be prefixed with the Jira id ([WINC-803](https://issues.redhat.com//browse/WINC-803)).

Also, there is a typo in the commit message: s/rlease/release

Comment (jrvaldes) [reply bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:11]: I suggest using `[bundle]` as the sub-component in the first line of the commit

File: bundle/tests/scorecard/config.yaml [hunk 3]
```diff
@@ -12,59 +12,38 @@ stages:
...
-      spec:
-        mountPath: {}
   - entrypoint:
     - scorecard-test
     - olm-bundle-validation
     image: quay.io/operator-framework/scorecard-test:v1.5.0
     labels:
       suite: olm
       test: olm-bundle-validation-test
-    storage:
```
Comment (jrvaldes) [inline bundle/tests/scorecard/config.yaml:25]: @selansen  These changes are not related and should be reverted.

File: config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -11,8 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
-    operators.openshift.io/valid-subscription: '["Red Hat OpenShift support for Windows
-      Containers"]'
```
Comment (jrvaldes) [inline config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:15]: Revert

File: config/rbac/role.yaml [hunk 5]
```diff
@@ -189,11 +197,3 @@ rules:
   - securitycontextconstraints
   verbs:
   - use
-- apiGroups:
```
Comment (jrvaldes) [inline config/rbac/role.yaml:192]: Revert

File: bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 6]
```diff
@@ -29,12 +29,13 @@ spec:
...
-    production clusters should acquire a subscription before attempting to install this operator. Users without a subscription
-    can try the community operator, a distribution which lacks official support.
+    Usage of this operator requires an OpenShift Container Platform for Windows Containers subscription. Users looking to
+    deploy Windows containers workloads in production clusters should acquire a subscription before attempting to
+    install this operator. Users without a subscription can try the community operator, a distribution which lacks
+    official support.
 
     ### Pre-requisites
-    * A Red Hat OpenShift subscription
+    * A Red Hat OpenShift Container Platform for Windows Containers subscription
```
Comment (saifshaikh48) [inline bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml:38]: Would it help to add a link to the support page here: https://access.redhat.com/support/policy/updates/openshift#windows
